### PR TITLE
Front-End for Documents

### PR DESCRIPTION
**Pull request by karissademi**

### CODE REVIEW

File: css/scss/_documents.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+       i{
+         padding-right:10px;
+      }
+      span::after{
+        content: ', ';
+      }
+      span:last-child::after{
+        content: '';
+      }
+    }//.doc-type
```
Comment (atogle) [inline css/scss/_documents.scss:19]: Are these closing comments helpful to anyone? I generally find them to be cluttery and inconsistently applied.

File: css/scss/base.scss [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
-@import 'news';
+
 @import 'departments';
+@import 'documents';
+@import 'home';
+@import 'news';
+
 
 .site-header {
     color: $white;
```
Comment (atogle) [inline css/scss/base.scss]: Consistent indentation

File: single-document.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+get right to the good stuff. */ ?>
+
+<div class="skip-link screen-reader-text">
+		<a href="#content" title="<?php esc_attr_e( 'Skip to content', 'twentyten' ); ?>">
+		<?php _e( 'Skip to content', 'twentyten' ); ?></a>
+</div>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class('row document'); ?>>
+<?php
+  	//loop for our regularly scheduled content
```
Comment (atogle) [inline single-document.php]: Inconsistent whitespace here (and elsewhere). What is best practice for PHP? Tabs vs spaces? 2 vs 4?

File: css/scss/_documents.scss (hunk 1 above)
Comment (karissademi) [inline css/scss/_documents.scss:19]: I like them to help clarify complex nesting.

File: single-document.php (hunk 3 above)
Comment (karissademi) [inline single-document.php]: I prefer one tab. Some of this is due to editor switching and little time to go back for clean up.

**Pull request merged**